helm-swoop.el

```diff
@@ -150,6 +150,15 @@
     (switch-to-buffer $buf))
   "Change the way to split window only when `helm-swoop' is calling")
 
+(defcustom helm-swoop-goto-line-action-hook nil
```

Comment:
For consistency, this should probably be called `helm-swoop-after-goto-line-action-hook`.  :)